ceph.spec.in

```diff
@@ -198,6 +197,7 @@ BuildRequires:	yasm
 BuildRequires:  librabbitmq-devel
 %endif
 %if 0%{with make_check}
+BuildRequires:	libuuid-devel
 BuildRequires:  jq
```

Comment:
though it may not be obvious at first glance, the lines in this conditional block are trying to be in alphabetical order ;-) could you move "libuuid-devel" down just a little bit more?

Reply:
sure. updated and repushed.